Client/Frontend/Browser/URLBarView.swift

```diff
@@ -424,12 +424,10 @@ class URLBarView: UIView {
             // Clear any existing text, focus the field, then set the actual pasted text.
             // This avoids highlighting all of the text.
             locationTextField.text = ""
-            locationTextField.becomeFirstResponder()
```

Comment:
Actually it looks like any tests that involve tapping the url bar by running:
```
        tester().tapViewWithAccessibilityIdentifier("url")
```
Seem to be failing on master at the moment. Would moving the if statement from above into this dispatch_async call work? I'll also see what's going on with these tests.